### ISSUE TITLE
NotificationItem refers to css with relative path that breaks as library

### ISSUE THREAD

**Issue by jhegg** [Tripwire/octagon#28]
In my app, I have the following:

```js
import NotificationItem from 'react-octagon/lib/components/NotificationItem'

export default class Thingy extends Component {
  render () {
    return (
      <NotificationItem notification={someNotifications[0]} data={[]} />
    )
  }
}
```

But it appears that `NotificationItem`'s css is in a relative path that works for the storybook, but not when using octagon as a library. I'm not sure if this is related to our discussion for https://github.com/Tripwire/octagon/issues/23, so I'm filing it as a separate issue.

```
Failed to compile.

Error in /data/src/octagon/lib/components/NotificationItem.js
Module not found: ../styles/components/notification-item.css in /data/src/octagon/lib/components

 @ /data/src/octagon/lib/components/NotificationItem.js 17:0-53
```

**Comment by eddier** [comment]
They are indeed related - we'll change the build process to deal with the CSS issue that is causing this.

**Comment by jhegg** [comment]
Sweet, thanks!

**Comment by eddier** [comment]
Can you see try the fonts branch again and see if the revised build step helps unblock you?  Be sure to run yarn install as a new package is added.

**Comment by jhegg** [comment]
@eddier the latest changes to the fonts branch indeed fixes this problem, awesome, thanks!

**Comment by cdaringe** [comment]
closed by #25
